test/integration/test_connection.yml

```diff
@@ -0,0 +1,35 @@
+- hosts: all
+  gather_facts: no
+  serial: 1
+  tasks:
+
```

Comment:
we might want to avoid ssh hosts by default as that setup is not guaranteed for the tests.